torch/nn/modules/linear.py

```diff
@@ -84,11 +84,14 @@ def __init__(self, in_features: int, out_features: int, bias: bool = True) -> No
         self.reset_parameters()
 
     def reset_parameters(self) -> None:
-        init.kaiming_uniform_(self.weight, a=math.sqrt(5))
-        if self.bias is not None:
-            fan_in, _ = init._calculate_fan_in_and_fan_out(self.weight)
-            bound = 1 / math.sqrt(fan_in)
-            init.uniform_(self.bias, -bound, bound)
+        if 0 not in self.weight.shape:
+            init.kaiming_uniform_(self.weight, a=math.sqrt(5))
+            if self.bias is not None:
+                fan_in, _ = init._calculate_fan_in_and_fan_out(self.weight)
+                bound = 1 / math.sqrt(fan_in)
```

Comment:
Alternatively, this solves the bias issue in a simpler way:
```python
bound = 1 / math.sqrt(fan_in) if fan_in > 0 else 0.
```

Reply:
Much better. Fixed in https://github.com/pytorch/pytorch/pull/56505/commits/9f86953b32fea04694eccf2b0d711820d05a0fd8